web-client/components/DepositConfirmationView.js

```diff
@@ -1,15 +1,19 @@
+
+import PropTypes from 'prop-types';
 import { ContentWrapper } from "./ContentWrapper";
 import NFTReviewScreenCard from "./NFTReviewScreenCard";
 import ViewWrapper from "./ViewWrapper";
 import React, { useState, useEffect } from "react";
 import Button from "./Button";
 
-function DepositConfirmationView(props) {
-    if (props.nftsToTrash != null) {
+const DepositConfirmationView = props => {
```

Comment:
I also changed all of the components from:
`function ComponentName(props) {`
to
`const ComponentName = props => {`
because thats more "standard" now to you the arrow notation 🤷‍♀️